### PR TITLE
Post-apply refactor

### PR DESCRIPTION
**Pull request by nickreynolds**
- refactor core to pull types out into `@joincivil/typescript-types` and util functions into `@joincivil/utils` for better reusability across packages
- delete nrsignup after application TX confirms, load listing data from graphql

### CODE REVIEW

File: packages/utils/src/gqlConstants.ts [hunk 1]
```diff
@@ -0,0 +1,570 @@
+import { pollHelpers } from "./coreHelpers";
```
[the remark this answers is not in the document]
Comment (nickreynolds) [reply packages/utils/src/gqlConstants.ts]: I'll split it up into "Queries" and "Transforms"

[the remark this answers is not in the document]
Comment (nickreynolds) [reply packages/utils/src/gqlConstants.ts]: and put it in a `gql` folder, yeah

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.3 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**